packages/app/src/app/pages/Dashboard/Content/routes/MyContributions/index.tsx

```diff
@@ -15,13 +16,18 @@ export const MyContributionsPage = () => {
     activeTeam,
     dashboard: { contributions },
   } = useAppState();
+  const { browser } = useEffects();
 
   React.useEffect(() => {
     actions.dashboard.getContributionBranches();
   }, [actions.dashboard]);
 
   const pageType: PageTypes = 'my-contributions';
 
+  const isNotificationDismissed = browser.storage.get(
```

Comment:
would it be possible to move these checks inside the component and render null when the notification has been dismissed? reduces the lines of code and the times browser.storage is called in the code

Reply:
Great idea, especially since I was already doing this 😄